oauthlib/oauth2/rfc6749/clients/backend_application.py

```diff
@@ -29,11 +29,11 @@ class BackendApplicationClient(Client):
     Since the client authentication is used as the authorization grant,
     no additional authorization request is needed.
     """
-    
+
     grant_type = 'client_credentials'
-    
+
     def prepare_request_body(self, body='', scope=None,
-                             include_client_id=None, **kwargs):
+                             include_client_id=True, **kwargs):
```

Comment:
Shouldn't be default to `False` to be backward compatible ?

Reply:
I agree.

Reply:
Yes, it should.